satellite/satellitedb/orders.go

```diff
@@ -214,3 +218,155 @@ func (db *ordersDB) UnuseSerialNumber(ctx context.Context, serialNumber storj.Se
 	_, err = db.db.ExecContext(ctx, db.db.Rebind(statement), storageNodeID.Bytes(), serialNumber.Bytes())
 	return err
 }
+
+// ProcessOrders take a list of order requests and "settles" them in one transaction
+func (db *ordersDB) ProcessOrders(ctx context.Context, requests []*orders.ProcessOrderRequest) (responses []*pb.SettlementResponse, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	if len(requests) == 0 {
+		return []*pb.SettlementResponse{}, err
```

Comment:
We should just return `nil, err` instead of `[]*pb.SettlementResponse{}, err` everywhere.

Reply:
Updated